src/store/reducers/dform/initial/styles.json

```diff
@@ -2976,6 +2976,72 @@
             }
           ]
         },
+        {
+          "class": "block",
+          "label": "WIDEL TEST",
+          "children":[
+            {
+              "field": "widelTest.positive",
+              "label": "Positive",
+              "class": "check"
+            },
+            {
+              "field": "widelTest.negative",
+              "label": "Negative",
+              "class": "check"
+            }
+          ]
+        },
+        {
+          "class": "block",
+          "show": "widelTest.positive",
+          "children":[
+            {
+              "field":"widelTest.Positive_SalO",
+              "label": "Sal. Typhi O",
+              "class": "radio",
```

Comment:
In contrast to the above, I think `"class": "check"` is suitable for here because this field has only one option and it seems the field should be boolean (true or false).
And users cannot "cancel" the input of this field if `"class": "radio"` is used.

Reply:
If it is true, and we click again on it will it be false?